data/profiles/demo.yaml

```diff
@@ -1,67 +1,60 @@
 apiVersion: install.istio.io/v1alpha2
```

Comment:
if we are changing the API don't we need to increment the api version?

Reply:
Actually this is a brand new API - just updated PR to reflect that.